rootfs/etc/nginx/lua/monitor.lua

```diff
@@ -61,11 +63,19 @@ local function flush(premature)
   clear_tab(metrics_batch)
   metrics_count = 0
 
-  local payload, err = cjson.encode(current_metrics_batch)
-  if not payload then
-    ngx.log(ngx.ERR, "error while encoding metrics: ", err)
-    return
+  local request_metrics = {}
+  table.insert(request_metrics, "[")
+  local item = ""
+  for i in pairs(current_metrics_batch) do
```

Comment:
Use `ipairs` is better if `current_metrics_batch` is an array-like table.

Reply:
done.